packages/esm-forms-dashboard-app/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { defineConfigSchema, getAsyncLifecycle } from '@openmrs/esm-framework';
+import { configSchema } from './config-schema';
+
+const importTranslation = require.context('../translations', false, /.json$/, 'lazy');
+
+const backendDependencies = {
+  'webservices.rest': '^2.2.0',
+  fhir2: '^1.2.0',
+};
+
+function setupOpenMRS() {
+  const moduleName = '@openmrs/esm-forms-dashboard-app';
+
+  const options = {
+    featureName: 'forms-dashboard',
+    moduleName,
+  };
+
+  defineConfigSchema(moduleName, configSchema);
+
+  return {
+    extensions: [
+      {
+        name: 'form-action-menu-item',
+        slot: 'action-menu-items-slot',
+        load: getAsyncLifecycle(
+          () => import('./clinical-form-action-button/clinical-form-action-button.component'),
+          options,
+        ),
+        order: 2,
+      },
+      {
+        name: 'forms-dashboard-workspace',
```

Comment:
It would actually be better to use the `registerWorkspace` function from patient-common-lib. See, e.g., [esm-patient-forms-app](https://github.com/openmrs/openmrs-esm-patient-chart/blob/9e3c013770b9fdd641e9a7efdc1a5ab63193fb07/packages/esm-patient-forms-app/src/index.ts)